css/modules.css

```diff
@@ -0,0 +1,41 @@
+#nav-menu {
+  display: none;
+  padding: 0;
+  list-style: none;
+  clear: both;
+}
+
+.icon-menu:before {
+  content:"\2630";
+  float: right;
+  font-size: 3em;
+  margin-top: .4em;
+}
+
+.nav-li {
+  margin-right: .4em;
+  font-size: 1.5em;
+  padding: .5em .2em;
+  border-bottom: 1px solid #ccc;
+}
+
+@media (min-width: 800px) {
+  #nav-menu {
+    display: inherit;
+    text-align: center;
+  }
+}
+
+@media (min-width: 800px) {
+.nav-li {
+  display: inline;
+  margin-left: 2em;
+  margin-right: 2em;
+  }
+}
+
+@media (min-width: 800px) {
+  .icon-menu:before {
+  display: none;
+  }
+}
```

Comment:
Great job on the media queries - since they are all the same 'min-width: 800px' you can group them all into the same/single query for cleaner code.